app/views/recipies/show.html.erb

```diff
@@ -0,0 +1,54 @@
+
+<div class="flex flex-col justify-center items-center pt-9">
+<h1 class="text-3xl text-bold"><%= @recipe.name%></h1>
+<div class="flex flex-col w-1/2 gap-3 text-lg">
+<div class="flex justify-between">
+<p>Preparation time: <%= @recipe.preparation_time%></p>
+<div class="self-end">
+ <% if @recipe.public %>
+    <span class="text-green-500 p-2 px-4">Public</span>
+<% else %>
+     <span class="text-red-500 p-2 px-4">Private</span>
+<% end %>
+</div>
+</div>
+<p>Cooking time: <%= @recipe.cooking_time%></p>
+<p><%= @recipe.description%></p>
```

Comment:
- [ ] Please make sure to add a toggle button on the recipe details page that allows users for the recipe to be made public or private as requested in the project requirements.

![image](https://github.com/abeltsew/recipe-app/assets/83803414/49515d4f-c298-4b43-8658-c3130b91eaec)

Reply:
Hi @thecodechaser ,

Thanks for the reviewer.  
Just add the feature,

I think we made a cool implementation of it, please check it out!!